### DISCUSSION TITLE
[Security] Galacticraft.team link in project description leads to malicious website(s)

### DISCUSSION THREAD

**Discussion by Vetpetmon** [TeamGalacticraft/Galacticraft#263] [Meta]
The link given in the repo's "About" section leads to a cycle of randomly-selected malware domains, some of which managed to bypass my uBlock extension.

I've ran into a risky chrome extension download page the first time I clicked it.
Second click linked me to this: 
![image](https://user-images.githubusercontent.com/10699805/233857992-c33881c0-953a-4934-bb51-f97d19005ea5.png)
Third click linked me to here: 
![image](https://user-images.githubusercontent.com/10699805/233858012-7633cb06-8696-45f5-b758-67c684cdad5a.png)

You may want to either reclaim the URL or delete it from the About section for user safety.

**Comment by mMONTAGEe** [comment]
Team is aware of this iirc
